CHANGELOG.md

```diff
@@ -53,6 +53,7 @@
   - Treat an empty `subset` (e.g. `[]`) as if it specified all columns instead of no columns.
   - Raise a `TypeError` for a scalar `subset` instead of filtering on just that column.
   - Raise a `ValueError` for a `subset` of type `pandas.Index` instead of filtering on the columns in the index.
+- Disable creation of scoped read only table to mitigate dynamic pivot unfound table issue in notebook env.
```

Comment:
I would suggest we describe the kind of error this would bubble earlier. For example 
```suggestion
- Disable creation of scoped read only table to mitigate `TableNotFoundError` when using dynamic pivot in notebook environment.
```

Reply:
udpated